Library/Homebrew/extend/ARGV.rb

```diff
@@ -268,6 +268,10 @@ def fetch_head?
     include? "--fetch-HEAD"
   end
 
+  def ignore_installed_specs?
+    include? "--ignore-installed-specs"
```

Comment:
I think the idea of the option is basically --use-old-behavior-before-pr-2191.

I think one issue we have with allowing, for example, a devel spec to fulfill a dependency is that we don't then force any reverse dependencies to be built from source, and will still pour bottles that may be specifically linked to the stable spec. So such an option would be a --get-me-out-of-linkage-hell-please-by-using-the-ones-that-actually-go-with-the-bottles-you're-pouring. Of course, the right answer is not to pour such bottles in the first place, and trigger the appropriate source builds.